CONTRIBUTING.md

```diff
@@ -149,6 +149,16 @@ git push
 
 **IMPORTANT**: By submitting a patch, you agree to the [License](#license).
 
+### Privacy Notice
+
+We would like to make you aware that contributing to OpenTTD via git will permanently store the name and email address you provide as well as the actual commit and the time and date you made it inside git's version history.
+
+This is inevitable, because it is a main feature git. If you are concerned about your privacy, we strongly recommend to use we strongly recommend to use "Anonymous &lt;anonymous@openttd.org&gt;" as the git commit author. We might refuse anonymous contributions if malicious intent is suspected.
+
+Please note that the EU GDPR right to have this information removed later cannot be honored as the OpenTTD project has the legitimate interest to keep a permanent record of the contributor identity, once given, for copyright verification and to provide proof should a malicious commit be made.
```

Comment:
Please note that the [EU GDPR](https://www.eugdpr.org/key-changes.html) "right to be forgotten" information cannot ..
Possibly a better way of phrasing?

Reply:
+1 it's not a refusal to honour the right, it's arguing that there's a legitimate reasons for retention.  IANAL, but I am happy to help with GDPR stuff.